frontend/src/components/managePermissions/table/DelegatedAdminTable.vue

```diff
@@ -0,0 +1,137 @@
+        value: null,
+        matchMode: FilterMatchMode.CONTAINS,
+    },
+    'role.role_name': {
+        value: null,
+        matchMode: FilterMatchMode.CONTAINS,
+    },
+    'role.parent_role.role_name': {
+        value: null,
+        matchMode: FilterMatchMode.CONTAINS,
+    },
+    'role.client_number.forest_client_number': {
+        value: null,
+        matchMode: FilterMatchMode.CONTAINS,
+    },
+});
+
+const delegatedAdminSearchChange = (newvalue: string) => {
+    delegatedAdminFilters.value.global.value = newvalue;
+};
+</script>
+
+<template>
+    <!-- Hidden until functionality is available
+    <ConfirmDialog group="deleteAdmin">
+        <template #message>
+            <ConfirmDialogtext
+                :userName=""
+                :role=""
+            />
+        </template>
+    </ConfirmDialog> -->
+    <div class="data-table-container">
+        <div class="custom-data-table">
+            <DataTableHeader
+                btnLabel="Create delegated admin"
+                :btnRoute="'#'"
+                :filter="delegatedAdminFilters['global'].value"
+                @change="delegatedAdminSearchChange"
+            />
+            <DataTable
+                v-model:filters="delegatedAdminFilters"
+                :value="props.delegatedAdmins"
+                paginator
+                :rows="50"
+                :rowsPerPageOptions="TABLE_ROWS_PER_PAGE"
+                filterDisplay="menu"
+                :loading="props.loading"
+                :globalFilterFields="[
+                    'user.user_name',
+                    'user.user_type.description',
+                    'role.role_name.role_name',
+                    'role.parent_role.role_name',
+                    'role.client_number.forest_client_number',
+                ]"
+                :paginatorTemplate="TABLE_PAGINATOR_TEMPLATE"
+                :currentPageReportTemplate="TABLE_CURRENT_PAGE_REPORT_TEMPLATE"
+                stripedRows
+            >
+                <template #empty> No user found. </template>
+                <template #loading> Loading users data. Please wait. </template>
+                <Column
+                    header="User Name"
+                    field="user.user_name"
+                    sortable
+                >
+                    <template #body="{ data }">
+                        <span>
+                            {{ data.user.user_name }}
+                        </span>
+                    </template>
+                </Column>
+                <Column
+                    field="user.user_type.description"
+                    header="Domain"
+                    sortable
+                ></Column>
+                <Column
+                    field="role.client_number.forest_client_number"
+                    header="Client ID"
+                    sortable
+                >
+                </Column>
+                <Column
+                    field="role.parent_role.role_name"
```

Comment:
Hi Jota, we have to fix here, so when the user is delegated for managing a concrete role like "FOM_REVIEWER", it's not displaying currently. So you might want to check here, if has parent role, then you display role.parent_role.role_name, if not, just display role.role_name

<img width="1110" alt="image" src="https://github.com/bcgov/nr-forests-access-management/assets/23131735/e8c07b34-2f12-4c18-af16-9cae041610ca">

Reply:
I did noticed that, I'll do the check. Thank you!